registration-processor/registration-processor-rest-client/src/main/java/io/mosip/registration/processor/rest/client/utils/RestApiClient.java

```diff
@@ -66,6 +67,12 @@ public class RestApiClient {
 	/** The logger. */
 	private final Logger logger = RegProcessorLogger.getLogger(RestApiClient.class);
 
+	@Value("${registration.processor.resttemplate.max.connection.per.route:20}")
```

Comment:
Can you call this as registration.processor.httpclient.connections.max.per.host instead of resttemplate?